### ISSUE TITLE
Missing 'zip', 'unique-device-id' and 'mobile-accessibility'.

### ISSUE THREAD

**Issue by henkkelder** [danielsogl/awesome-cordova-plugins#3937]
I'm submitting a ... (check one with "x")
[ ] bug report
[x ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://ionicworldwide.herokuapp.com/

Current behavior:
Plugins missing
- @ionic-native/zip
- @ionic-native/mobile-accessibility
- @ionic-native/unique-device-id
All three of them are mentioned at https://ionicframework.com/docs/native/community

Expected behavior:
Seamless migration to awesome-cordova-plugins. Now it breaks my apps.

Steps to reproduce:
Try to install these from @awesome-cordova-plugins

plugin not found

**Comment by danielsogl** [comment]
The Ionic team removed all outdated and unmaintained plugins from this repo. You should replace your cordova based plugins with capacitor plugins if possible.
